File: sentimental-mario-less/mario.py
from cs50 import get_int


while True:
	n = get_int("Height: ")
	if n > 0 and n <= 8:
		break



for i in range(n):

    for j in range(n):

        if j + i < n - 1:
           print(" ", end="")
        else:
            print("#", end="")
    print("  ", end="")

    for j in range(n - 1):
        if j + i < n - 1:
           print("", end="")
        else:
            print("#", end="")

    print("#", end="")

    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
do
{
    n = get_int("Height: ");
}
while (n > 8 || n <1);


for(int i = 0; i < n; i++)
{
    for (int j = 0; j < n; j++)
    {
        if (j + i < n - 1)
        {
            printf(" ");
        }
        else
        {
            printf("#");
        }
    }
    printf("\n");
}
}